### ISSUE TITLE
Reference Error: Highcharts is not defined (in some environments) 

### ISSUE THREAD

**Issue by dominikchudy** [highcharts/highcharts#14059]
When using `series.centerInCategory` property, in some environments (in this case in Vue.js) we get the Reference Error `Highcharts is not defined.` It is because we refer to the global `Highcharts` object (which is not available there) instead of `H`.

#### Live demo:
https://codesandbox.io/s/holy-leftpad-g2jfq?file=/src/components/BarChart.vue

```
Highcharts.objectEach(this.yAxis.stacking && this.yAxis.stacking.stacks, function(
  stack
)
```

As a workaround we can replace `Highcharts` with `H`:
```
H.objectEach(this.yAxis.stacking && this.yAxis.stacking.stacks, function(
  stack
)
```
#### Working demo
https://codesandbox.io/s/thirsty-swartz-lvegv?file=/src/components/BarChart.vue

#### Product version
Highcharts 8.1.2

#### Affected browser(s)
All

FYI @bre1470

**Comment by bre1470** [comment]
@dominikchudy 
Hi! Thanks for the issue report. I see in your failing demo no indication, that Highcharts itself is failing. This looks like an import problem to me. Could you solve the issue?

@Denyllon 
Highcharts-Vue is not tested with v8. (see devDependencies) Might this be a problem?

**Comment by Denyllon** [comment]
@bre1470 @dominikchudy It does look like the problem is fixed in `v8.2.0` since the https://github.com/highcharts/highcharts/commit/3ba09e661f776f9e4e42414edae9d19802937f53 commit. There was the `Highcharts` reference, which indicated on a global object which was not available there.
While using `v8.2.0` everything works well: https://codesandbox.io/s/holy-meadow-ew8dt
